wagtail/admin/views/mixins.py

```diff
@@ -224,7 +225,7 @@ def get_heading(self, queryset, field):
         if heading_override:
             return force_str(heading_override)
         try:
-            return force_str(queryset.model._meta.get_field(field).verbose_name.title())
+            return force_str(label_for_field(field, queryset.model)).title()
```

Comment:
I wonder if we should use `capfirst()` here instead of `title()` so that we don't make unnecessary lowercasing of the value, e.g. if the field has a `verbose_name` of `response data JSON` it becomes `Response data JSON` instead of `Response Data Json`. Similar issue with the column was #10434.

Reply:
Without this change, we would still have a discrepancy between the export headers and listing headers, so I added this in 391cdb5.